matlab-hello-world-model/README.md

```diff
@@ -0,0 +1,106 @@
+   This information will be shown to other users who may wish to 
+   use your Model.
+- _[dataset.csv](./dataset.csv)_ - an example csv file to run
+   the Model with, on DAFNI this file will be provided by a 
+   dataset.
+- _[helloWorld.m](./helloWorld.m)_ - the Matlab Model code
+- _[run_model.sh](./src/run_model.sh)_ - a script that copies
+   the input dataset files from DAFNI's data directory to the 
+   folder the Model expects the files to be in, then runs the 
+   Model and then copies the output file to DAFNI's data directory
+   so DAFNI can find it.
+
+As this Model uses Matlab the process required to build a Docker
+image that can run the Model is more complicated than for the 
+other example Models. In fact the process we use is to generate
+a Docker image from Matlab itself and then use that as a base
+for the Dockerfile in this folder.
+
+To start with you will need to have a Linux machine with both
+Matlab and Docker installed. The reason you need a Linux machine
+is because the Matlab commands needed to produce the Docker image
+only work on Linux, as you can see from their
+[documentation](https://uk.mathworks.com/help/compiler/package-matlab-standalone-applications-into-docker-images.html).
+While you could use this documentation to work out how to dockerise
+this Model for yourself I would recommend continuing on with this
+README as we have already tweaked the commands to work for this
+case.
+
+- Okay now you have a Linux machine with Matlab and Docker
+  installed, copy across `helloWorld.m` into your Matlab workspace.
+- We need to make this Matlab script into a standalone 
+  application, in order to do this run the following in your Matlab
+  console `res = compiler.build.standaloneApplication('helloWorld.m')`
+- Now we need to create a DockerOptions object, to do so we need 
+  to pass in the object from the previous step and the name you 
+  want to use for this intermediate Docker image using:
+  `opts = compiler.package.DockerOptions(res,'ImageName','hello-world')`
+- To actually create the Docker image from Matlab we need to pass
+  the objects produced from the previous two steps into Matlab's
+  Docker compiler using: `compiler.package.docker(res, 'Options', opts)`.
+  When this command finishes running you can confirm that
+  the Docker image has been created properly by running 
+  `docker image ls` in a terminal (not your Matlab console). You 
+  should see a `hello-world` image listed as well as another image
+  or two that Matlab has downloaded for you, you can ignore these
+  extra images we only need the `hello-world` image.
+- At this point we no longer need Matlab, navigate the terminal you
+  opened to check the `hello-world` image exists, or open a new
+  terminal and navigate it, to your local copy of this folder.
+- Once you have your terminal pointing at this folder then you can
+  build the Docker image for this Model using 
+  ```bash
+  docker build -t hello-world-matlab .
+  ```
```

Comment:
This is a bit confusing. 

I'd just start out by saying "We no longer need Matlab and all future commands should be executed in a new terminal" or something rather than introducing the new terminal and then referring back to it later.